CMakeLists.txt

```diff
@@ -15,7 +15,8 @@ option(BENCHMARK_ENABLE_TESTING "Enable testing of the benchmark library." ON)
 option(BENCHMARK_ENABLE_EXCEPTIONS "Enable the use of exceptions in the benchmark library." ON)
 option(BENCHMARK_ENABLE_LTO "Enable link time optimisation of the benchmark library." OFF)
 option(BENCHMARK_USE_LIBCXX "Build and test using libc++ as the standard library." OFF)
-option(BENCHMARK_BUILD_32_BITS "Build a 32 bit version of the library" OFF)
+option(BENCHMARK_BUILD_32_BITS "Build a 32 bit version of the library." OFF)
+option(INSTALL_BENCHMARK "Enable installation of benchmark. (Projects embedding benchmark may want to turn this OFF.)" ON)
```

Comment:
`s/INSTALL_BENCHMARK/BENCHMARK_ENABLE_INSTALL/`

for consistency and so it's clear that it's not doing the actual installation.

Reply:
Thanks - will do.